src/components/actividades/ListadoActividades.js

```diff
@@ -120,6 +131,12 @@ export default function ListadoActividades() {
 
   return (
     <>
+      {mostrarAlerta ? (
+        <Alert severity="success">
+          El link a {actividadCopiada} fue copiado al portapeles
+        </Alert>
+      ) : null}
```

Comment:
Si ya tienen un estado, para que hacen un ternario? 🤔 
```suggestion
      {mostrarAlerta &&
        <Alert severity="success">
          El link a {actividadCopiada} fue copiado al portapeles
        </Alert>
      }
```